third_party/xla_client/stablehlo_helper.cc

```diff
@@ -0,0 +1,44 @@
+#include "third_party/xla_client/stablehlo_helper.h"
+
+#include <iostream>
+
+#include "mlir/IR/Verifier.h"       // from @llvm-project
+#include "mlir/Pass/PassManager.h"  // from @llvm-project
+#include "tensorflow/compiler/xla/mlir_hlo/mhlo/transforms/passes.h"
+#include "tensorflow/compiler/xla/translate/hlo_to_mhlo/hlo_to_mlir_hlo.h"
+#include "tensorflow/compiler/xla/translate/mhlo_to_hlo/mlir_hlo_to_hlo.h"
+#include "third_party/xla_client/debug_macros.h"
+#include "third_party/xla_client/xla_util.h"
+
+namespace xla {
+
+static void hlo_mhlo_helper(const HloModuleProto* proto,
+                            mlir::ModuleOp* mlir_module) {
+  auto status = ConvertHloToMlirHlo(*mlir_module, proto,
+                                    /*import_all_computations*/ false);
+  XLA_CHECK(status.ok()) << "HLO -> MHLO conversion failed.";
+  XLA_CHECK(mlir::verify(*mlir_module).succeeded())
+      << "MHLO from HLO->MHLO is not leagal.";
+}
+
+static void mhlo_stablehlo_helper(mlir::ModuleOp* mlir_module,
```

Comment:
nit, `mhlo_to_stablehlo_helper `

Reply:
Will update

Reply:
another nit: shouldn't our cpp code be camelcase?

Reply:
Yes, good catch, just realized that and will update.